test/test_nn_ops.cpp

```diff
@@ -106,46 +106,198 @@ TEST(NNOps, BiasAddGrad) {
   opexecuter_4D.RunTest();
 }
 
-TEST(NNOps, Conv2DBackpropFilterNHWC) {
+// TF does not support NCHW kernels
+// To test NCHW data format
+// Create graph with inputs in NCHW format and execute on NGraph
+// Reshape the NCHW inputs to NHWC and run on TF
+// Compare the results
+TEST(tf_exec, Conv2DBackpropFilterNCHWSame) {
+  string padding_type = "SAME";
+  // Filter :[filter_height, filter_width, in_channels, out_channels]
+  initializer_list<int> filter_size_HWIO = {3, 3, 2, 2};
+  vector<DataType> output_datatypes = {DT_FLOAT};
+  vector<int> static_input_indexes = {1};
+
+  // Define scope for nGraph
+  // Data Format : NCHW
+  Scope ngraph_scope = Scope::NewRootScope();
+  vector<int64> input_size_NCHW = {1, 2, 7, 6};
+  Tensor input_data_NCHW(DT_FLOAT, TensorShape(input_size_NCHW));
+  AssignInputValuesAnchor(input_data_NCHW, -1.1f);
```

Comment:
Hi @shresthamalik , is there a reason you are using `AssignInputValuesAnchor()` here? I think it will be good to just use random values via AssignInputValuesRandom() and gives a (min,max) range.

Reply:
Yes, i forgot to update it to use Random. Will update this.